mkdirp/test.ts

```diff
@@ -0,0 +1,30 @@
+import { args, lstat, removeAll, FileInfo } from 'deno'
+import { test, assert } from 'https://deno.land/x/testing/testing.ts'
+import mkdirp from './mkdirp.ts'
+
+const root_dir: string = args[1] || './mkdirp_test_root_dir'
```

Comment:
Hrm.. Since this test will be run along with others, using `args[1]` isn't a good idea. 

It would be good to use makeTempDirSync instead to create a working directory https://github.com/denoland/deno/blob/68e009d1d8cdf60ac2957d6b7c56b3b447cbc81e/js/mkdir_test.ts#L12-L20